cmd/security-file-token-provider/res/token-config.json

```diff
@@ -175,5 +175,11 @@
         }
       }
     }
+  },
+  "rules-engine": {
+    "edgex_use_defaults": true
+  },
+  "ui": {
+    "edgex_use_defaults": true
```

Comment:
On the fence about adding these here verses using env var in comopses file when zero-trust is enabled.

Reply:
Sure. I had asked Bryon about how I would get tokens emitted and this was what he instructed me to do. However you think it's best

Reply:
Cool, I'll defer to his expertise. ;-)